inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/HtmlAnnotationEditor.java

```diff
@@ -155,9 +155,9 @@ public String renderHtml()
         }
 
         // Sort backwards
-        nodes.sort((a, b) -> {
-            return b.position - a.position;
-        });
+        nodes.sort((a, b) -> 
+             b.position - a.position
+        );  //30
```

Comment:
What does the //30 mean here?

Reply:
I added these comments to track my tasks. It does not say anything about the changes i made. I will remove them.

Reply:
I added these comments while tracking my tasks. They are not relevant to the changes that I made. I shall remove them.